src/registrar/tests/test_transition_domain_migrations.py

```diff
@@ -20,6 +20,9 @@
 
 from .common import MockSESClient, less_console_noise
 import boto3_mocking  # type: ignore
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
Remember to remove this, it seems like its just used with an added logger.debug

Reply:
I intentionally left the logger statements in.  Just in case someone needs to debug the unit test, they can remove the "with_less_noise" and run the test.